core/src/HIP/Kokkos_HIP_Space.cpp

```diff
@@ -575,7 +575,8 @@ namespace Experimental {
 
 int HIP::concurrency() {
   // FIXME_HIP
-  return 32 * 8 * 40;  // 81920 fiji and hawaii
+  // MI60: ThreadsPerComputeUnit*ComputeUnits/ShaderEngine*ShaderEngines)
+  return 16 * 4 * 1024;
```

Comment:
Is this right? Don't they have like 2536 threads or so per compute unit? (1024 is the limit per block or group, but you can have like 5 groups of 512 simultaneously or?)

Reply:
https://www.olcf.ornl.gov/wp-content/uploads/2019/10/ORNL_Application_Readiness_Workshop-AMD_GPU_Basics.pdf
Yeah its 40 wave fronts with 64 threads each per CU i.e. 2536 threads per CU.